mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/AddressBookScreenV2.tsx

```diff
@@ -299,72 +300,53 @@ export function AddressBookScreenV2 ({ route, navigation }: Props): JSX.Element
             />
           </View>
           <View style={tailwind('ml-3')}>
-            {isSearchFocus
-            ? (
-              <ThemedTouchableOpacityV2
-                onPress={() => {
-                  Keyboard.dismiss()
-                  setIsSearchFocus(false)
-                  setSearchString('')
-                }}
-                light={tailwind('bg-mono-light-v2-900')}
-                dark={tailwind('bg-mono-dark-v2-900')}
-                testID='cancel_search_button'
-                style={tailwind('flex h-10 w-10 flex-row items-center justify-center rounded-full')}
-              >
-                <ThemedIcon
-                  size={24}
-                  name='x'
-                  light={tailwind('text-mono-light-v2-00')}
-                  dark={tailwind('text-mono-dark-v2-00')}
-                  iconType='Feather'
-                />
-              </ThemedTouchableOpacityV2>
-            )
-            : (
-              <>
-                {activeButtonGroup === ButtonGroupTabKey.Whitelisted
-                  ? (
-                    <ThemedTouchableOpacityV2
-                      onPress={goToAddAddressForm}
-                      light={tailwind('bg-mono-light-v2-900')}
-                      dark={tailwind('bg-mono-dark-v2-900')}
-                      testID='add_new_address'
-                      style={tailwind('flex h-10 w-10 flex-row items-center justify-center rounded-full')}
-                    >
-                      <ThemedIcon
-                        size={24}
-                        name='plus'
-                        light={tailwind('text-mono-light-v2-00')}
-                        dark={tailwind('text-mono-dark-v2-00')}
-                        iconType='Feather'
-                      />
-                    </ThemedTouchableOpacityV2>
-                  )
-                  : <DiscoverWalletAddressV2 size={24} />}
-              </>
-            )}
+            {activeButtonGroup === ButtonGroupTabKey.Whitelisted
+              ? (
+                <ThemedTouchableOpacityV2
+                  onPress={goToAddAddressForm}
+                  light={tailwind('bg-mono-light-v2-900')}
+                  dark={tailwind('bg-mono-dark-v2-900')}
+                  testID='add_new_address'
+                  style={tailwind('flex h-10 w-10 flex-row items-center justify-center rounded-full')}
+                >
+                  <ThemedIcon
+                    size={24}
+                    name='plus'
+                    light={tailwind('text-mono-light-v2-00')}
+                    dark={tailwind('text-mono-dark-v2-00')}
+                    iconType='Feather'
+                  />
+                </ThemedTouchableOpacityV2>
+              )
+              : <DiscoverWalletAddressV2 size={24} />}
           </View>
         </View>
-        {isSearchFocus && (
-          <View style={tailwind('px-5 mt-8 mb-2')}>
+        {(isSearchFocus || searchString.trim().length !== 0) && (
```

Comment:
Can we add a searchString?. check? So in case it's null or undefined

Reply:
searchString will never going to be null or undefined, on clearing input we are setting it as empty string